### PR TITLE
added markdown escapes for _*~\

### PR DESCRIPTION
**Pull request by rvagg**
Fixes: #47

### CODE REVIEW

**Comment by rvagg** [comment]
Note that this doesn't depend on word boundaries. At least for GitHub it matters for some: test_test_ and  test\test\ are fine, test*test* test~test~ are not. Since it doesn't (and I'd assume other parsers) have a problem with extra unnecessary escaping then I think we should be fine going the extra mile.

File: commit-to-output.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const chalk   = require('chalk')
 
 function cleanMarkdown (txt) {
   // escape []<>
```
Comment (ChALkeR) [inline commit-to-output.js]: Update comment?

Comment (rvagg) [reply commit-to-output.js]: ta, fixed

**Comment by ChALkeR** [comment]
@rvagg Also, just noticed — what about backticks (`)? Escaping those won't harm, unless when in an inline code block — inside those, it doesn't help.

E.g.:
* Raw: ` x ` y ` z
* Escaped: \` x \` y \` z
* Inside inline code block: `\` x \` y \` z`

**Comment by rvagg** [comment]
well.. for Node changelogs we've relied on backticks to render code things and it's actually pretty common. Check out the changelog for 9.7.0 that's about to go out (rendered with this latest version of changelog-maker btw so it has extra escapes for `_`'s): https://github.com/nodejs/node/blob/v9.7.0-proposal/doc/changelogs/CHANGELOG_V9.md#9.7.0

**Pull request merged**